rollup.config.mjs

```diff
@@ -1,4 +1,4 @@
-import ts from "rollup-plugin-ts";
```

Comment:
`rollup-plugin-ts` has been deprecated and they recommend moving to `@rollup/plugin-typescript`